final_prep/README.md

```diff
@@ -45,30 +45,39 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+
+[Week 1] <img width="1257" alt="Screen Shot 2022-02-13 at 3 58 28 PM" src="https://user-images.githubusercontent.com/48334178/153779098-3207a456-c405-4d41-9070-2bb352a709cd.png">
+
 - `Add Week 2 Screenshot Here`
+
+[Week 2]<img width="1269" alt="Screen Shot 2022-02-13 at 3 58 39 PM" src="https://user-images.githubusercontent.com/48334178/153779137-ce6fe9bd-eaf7-491f-bd2f-55049099105e.png">
+
 - `Add Week 3 Screenshot Here`
 
+[Week 3]<img width="1267" alt="Screen Shot 2022-02-13 at 3 58 49 PM" src="https://user-images.githubusercontent.com/48334178/153779161-748d221b-a38d-4ce7-b10e-6672258a5f08.png">
+
+
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
```

Comment:
You could add a little more detail here. What are you hoping to get out of a mentor relationship?

Reply:
Also, that link is to the original copy of the doc. I found your forked copy, but just wanted you to know!